IdTag/style.css

```diff
@@ -0,0 +1,11 @@
+@import '../variables.css';
+
+.label {
```

Comment:
Heya @stevemdixon, @hharnisc  - not sure if the position should be absolute here. I think we'll be using this mostly to display over the thumbnail for ImagePosts. 🤔